src/apis/api.ts

```diff
@@ -41,3 +33,67 @@ export function patch<T>(...args: Parameters<typeof instance.patch>) {
 export function del<T>(...args: Parameters<typeof instance.delete>) {
   return instance.delete<T>(...args);
 }
+
+const fetchAccessToken = async () => {
+  const refreshToken = localStorage.getItem('refreshToken');
+
+  try {
+    const response = await axios.get<ApiResponseType<AccessTokenGetSuccess>>(
+      `${import.meta.env.VITE_APP_BASE_URL}/v1/user/token-refresh?refreshToken=${refreshToken}`
+    );
+
+    const accessToken = response.data.data.accessToken;
+    if (!accessToken) {
+      return null;
+    }
+    localStorage.setItem('accessToken', accessToken);
+    instance.defaults.headers.Authorization = `Bearer ${accessToken}`;
+    return accessToken;
+  } catch (error) {
+    console.error('토큰 재발급 실패:', error);
+
+    const errorData = (error as AxiosError)?.response?.data as ErrorType;
+
+    // 리프레시 토큰 만료 시. 로그아웃 처리
+    if (errorData.status === 40101) {
+      console.error('리프레시 토큰 만료:', errorData);
+      localStorage.removeItem('user');
+      localStorage.removeItem('accessToken');
+      localStorage.removeItem('refreshToken');
+      window.location.href = '/login';
+      return Promise.reject(error);
+    }
+  }
+};
```

Comment:
```suggestion
  } catch (error) {
  tokenError();
};

const tokenError = () => {
 console.error('토큰 재발급 실패:', error);

    const errorData = (error as AxiosError)?.response?.data as ErrorType;

    // 리프레시 토큰 만료 시. 로그아웃 처리
    if (errorData.status === 40101) {
      console.error('리프레시 토큰 만료:', errorData);
      clearLocalStorage();
      window.location.href = '/login';
      return Promise.reject(error);
    }
  }
};
}

const clearLocalStorage = () => {
localStorage.removeItem('user');
      localStorage.removeItem('accessToken');
      localStorage.removeItem('refreshToken');
}
```
가독성을 위해 에러 로직을 분리하는건 어떨까요?

Reply:
아 이거 유틸로 하나 뺄게요